snuba/query/columns.py

```diff
@@ -0,0 +1,169 @@
+from itertools import chain
```

Comment:
I do not expect this module to exist in this form for long. column_expr and similar functions are going to be embedded in column abstractions as soon as they are introduced, but we need to break it off from util to be able to make Query more structured, which is required to introduce Column abstractions, which is required to embed these functions into such abstractions.

As of now 100% cut/paste from util.